README.rst

```diff
@@ -42,4 +42,235 @@ Other code (in ``libcore``, ``lib``, etc.) should not be used by external users
+``isopen``                  boolean                 Is this an open codelist?
+``values``                  array*                  Values in the data that were not on the codelist
+``extension_codelist``      boolean                 Was the codelist added by an extension? (Not just modified by it).
+=========================== ======================= ============
+
+additional check object
+^^^^^^^^^^^^^^^^^^^^^^^
+
+=========================== ===================== ==============
+Property (key) name	    Type                  Value
+=========================== ===================== ==============
+``json_location``           string                A JSON Pointer to the data that was problematic e.g. ``releases/0/buyer``
+=========================== ===================== ==============
+
+
+conformance_errors
+^^^^^^^^^^^^^^^^^^
+
+=============================== ======================= =====
+Property (key) name	        Type                    Value
+=============================== ======================= =====
+``ocds_prefixes_bad_format``    array[array[string]]    An array of pairs of the bad ocid value, and the JSON Pointer to it, e.g. ``["MY-ID", "releases/0/ocid"]``
+``ocid_description``            string                  The descriptive text about ocids taken from the schema.
+``ocid_info_url``               string                  The url to the information about identifiers in the reference docs
+=============================== ======================= =====
+
+additional_fields
+^^^^^^^^^^^^^^^^^
+
+============================= ========= ==============
+Property (key) name	      Type      Value
+============================= ========= ==============
+``path``                      string    The path of the parent object, e.g. ``/publisher``
+``field``                     string    The JSON property name, e.g. ``myField``
+``usage_count``               integer   How many times this additional field appears
+============================= ========= ==============
+
+all_additional_fields
+^^^^^^^^^^^^^^^^^^^^^
+
+=================================== =========== ==============
+Property (key) name	            Type        Value
+=================================== =========== ==============
+``count``                           integer     How many times this additional field appears
+``examples``                        array*      An array of values for this field
+``root_additional_field``           boolean     Is this the first additional field we find descending into this bit of the schema? ie. is the parent in the schema?
+``additional_field_descendance``    object      Only appears if ``root_additional_field`` is true. A mapping from paths, to objects like those in all_additional_fields_, for each of the additional fields that can be found by descending into the data from this field.
```

Comment:
Isn't this description circular? ("like those in all_additional_fields_")

```suggestion
``additional_field_descendance``    object      Is only set if ``root_additional_field`` is true. A mapping from paths to objects, like those in all_additional_fields_, for each of the additional fields that can be found by descending into the data from this field.
```

Reply:
It's a recursive structure, except that `root_additional_field` is always false on the second level, so it doesn't get any further than that.

Reply:
Thanks, I've suggested new text in a new review.